Formula/orocos-kdl.rb

```diff
@@ -0,0 +1,33 @@
+class OrocosKdl < Formula
+  desc "Orocos Kinematics and Dynamics C++ library"
+  homepage "http://www.orocos.org/kdl"
+  url "https://github.com/orocos/orocos_kinematics_dynamics/archive/v1.3.1.tar.gz"
+  sha256 "aff361d2b4e2c6d30ae959308a124022eeef5dc5bea2ce779900f9b36b0537bd"
+
+  depends_on "cmake" => :build
+  depends_on "eigen"
+
+  def install
+    cd "orocos_kdl" do
+      system "cmake", ".", "-DEIGEN3_INCLUDE_DIR=#{Formula["eigen"].opt_include}/eigen3",
+                           *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    (testpath/"test.cpp").write <<~EOS
+      #include <kdl/frames.hpp>
+      int main()
+      {
+        using namespace KDL;
+        Vector v1(1.,0.,1.);
+        Vector v2(1.,0.,1.);
+        assert(v1==v2);
+        return 0;
+      }
+    EOS
+    system ENV.cxx, "test.cpp", "-I#{include}", "-L#{lib}", "-lorocos-kdl", "-o", "test"
```

Comment:
we're passed the 80th column

Reply:
Will fix but I'm not alone ;) `grep -e 'system ENV.cxx.*test' *.rb`

Reply:
Fixed but I followed the example in another formula.  Let me know if I should put each option on a separate line with the same indentation or if what I did is acceptable.  Fewer lines == betterer?